### PR TITLE
Fuel wallet development

### PR DESCRIPTION
**Pull request by luisburigo**

### CODE REVIEW

File: src/modules/vault/hooks/assets/useVaultAssets.ts [hunk 1]
```diff
@@ -113,6 +113,7 @@ function useVaultAssets(predicate?: Vault) {
 
       return (
         bn(bn.parseUnits(balance.amount!))
+          //@ts-ignore - TODO: minGasPrice was removed in fuels@0.88.1
           .sub(gasConfig?.minGasPrice || bn.parseUnits('0.00001'))
```
Comment (luisburigo) [inline src/modules/vault/hooks/assets/useVaultAssets.ts:117]: `minGasPrice` was removed in fuels@0.88.1

File: src/modules/transactions/services/methods.ts [hunk 2]
```diff
@@ -128,13 +128,11 @@ export class TransactionService {
       },
     });
 
-    const { gasPrice, usedFee, minFee } =
+    const { maxFee } =
       await vault.provider.getTransactionCost(transactionRequest);
```
Comment (luisburigo) [inline src/modules/transactions/services/methods.ts:132]: In local tests, the `maxFee` is the exact price of fee

Comment (guimroque) [reply src/modules/transactions/services/methods.ts:132]: use maxGasPerPredicate

File: src/modules/vault/hooks/assets/useVaultAssets.ts (hunk 1 above)
Comment (guimroque) [reply src/modules/vault/hooks/assets/useVaultAssets.ts]: use maxGasPerPredicate

**Comment by luisburigo** [comment]
In preview/production
image

In development
image

File: src/modules/vault/hooks/assets/useVaultAssets.ts [hunk 3]
```diff
@@ -113,7 +113,8 @@ function useVaultAssets(predicate?: Vault) {
 
       return (
         bn(bn.parseUnits(balance.amount!))
-          .sub(gasConfig?.minGasPrice || bn.parseUnits('0.00001'))
+          //@ts-ignore - TODO: minGasPrice was removed in fuels@0.88.1
```
Comment (luisburigo) [inline src/modules/vault/hooks/assets/useVaultAssets.ts:116]: remove comment

**Review by guimroque — APPROVED** [review]

**Pull request merged**